ccd-definition/CaseEvent/User/UserEvents-HWF-R2-CUI-nonprod.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "FULL_REMISSION_HWF",
+    "Name": "Full remission HwF - granted",
+    "Description": "Full remission HWF - granted",
+    "PreConditionState(s)": "PENDING_CASE_ISSUED",
```

Comment:
Add "HEARING_READINESS" in PreConditionStates as this will be case state for hearing fee.